Sources/PKYouboraAdsAdapter.swift

```diff
@@ -156,6 +156,7 @@ extension PKYouboraAdsAdapter {
                     guard let self = self else { return }
                     // Update ad info with the new loaded event
                     self.adInfo = event.adInfo
+                    self.plugin?.fireInit()
```

Comment:
@x-NR-x adLoaded is not too late?
we will not have the correct join time I am afraid

Reply:
Not from what I saw. 
'adsRequested' is performed upon prepare, and if the player was set as autoplay false I don't want the join time to start counting. 
Now that I am thinking about it, I need to change it, 'adLoaded' is not good for the case that the player is on autoplay false it is also early.

Reply:
This is the best place to put it for now.
Need to add a way to identify if the player is on autoplay or not.